docs/content/tutorials/oidc-custom-configuration.md

```diff
@@ -0,0 +1,199 @@
+---
+title: NGINX Ingress Controller and Open Service Mesh
+description: |
+  Use NGINX Ingress Controller with Open Service Mesh.
+weight: 1800
+doctypes: ["concept"]
+toc: true
+docs: "DOCS-1181"
+---
+
+# OIDC Custom Configuration
+
+The F5 NGINX Ingress Controller implements OpenID Connect (OIDC) using the NGINX OpenID Connect Reference implementation: [nginx-openid-connect](https://github.com/nginxinc/nginx-openid-connect).
+
+This guide will walk through how to customise and configure this default implementation.
+
+## Prerequisites
+This guide assumes that you have an F5 NGINX Ingress Controller deployed. If not, please follow the installation steps using either the [Manifest](https://docs.nginx.com/nginx-ingress-controller/installation/installation-with-manifests/) or [HELM](https://docs.nginx.com/nginx-ingress-controller/installation/installation-with-helm/) approach.
+
+To customise the NGINX OpenID Connect Reference implementation, we will need to:
```

Comment:
```suggestion
To customise the NGINX OpenID Connect Reference implementation, you will need to:
```
There's a few different instances of we vs you in this document: "we" is overly familiar with the reader and somewhat informal for imperative instructions.